commerce/src/index.d.ts

```diff
@@ -576,6 +576,7 @@ declare global {
                 wcsBufferDelay: number;
                 wcsBufferLimit: number;
                 wcsEnv: Environment;
+                domainSwitch: boolean;
```

Comment:
do we really need to introduce a new parameter with a confusing name:`domainSwitch` ?
I would like to avoid having to support temporarily introduced parameters for ever.
Can we just switch to this new domain?

Reply:
@yesil we need both domains to work so we can compare stuff from same domain